app/human-detector-app/screens/GroupScreen.tsx

```diff
@@ -1,23 +1,123 @@
 import * as React from 'react';
-import { StatusBar } from 'expo-status-bar';
-import { View, Text } from 'react-native';
+import { useState } from 'react';
+import { ScrollView, StyleSheet, Text, TouchableOpacity, View } from 'react-native';
+import Camera from '../classes/Camera';
+import CameraSettingsButton from '../components/CameraSettingsButton';
+import Group from '../classes/Group';
 
-export default function GroupScreen(){
-    
-}
+export default function GroupMenu({ navigation }: Props): React.ReactElement {
+  const groupOne: Group = new Group("AAAAAA's Group", '99');
+  const groupTwo: Group = new Group("BBBBB's Group", '725');
+  const groupThree: Group = new Group("CCCCC's Group", '400');
+
+  const [listOfCameras, setListOfCameras] = useState([groupOne, groupTwo, groupThree]);
```

Comment:
In the future, we might want to consider using [React Native Testing Library](https://www.npmjs.com/package/@testing-library/react-native) to test that our components render the way we expect them to (if we have the time to set it up)

Reply:
It was actually something that I was working on last quarter but I just stuck with the unit tests that we have rn for now